content/id/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,414 @@
+Container yang berbahaya dapat menggunakan kekurangan ini untuk menimpa konten biner `runc` dan
+akibatnya Container tersebut dapat menjalankan perintah yang sewenang-wenang pada sistem host dari Container tersebut.
+
+Silahkan merujuk pada [CVE-2019-5736](https://access.redhat.com/security/cve/cve-2019-5736) untuk informasi lebih lanjut tentang masalah ini.
+{{< /caution >}}
+
+### Penerapan
+
+{{< note >}}
+Dokumen ini ditulis untuk pengguna yang memasang CRI (Container Runtime Interface) pada sistem operasi Linux. Untuk sistem operasi yang lain, 
+silahkan cari dokumentasi khusus untuk platform kamu.
+
+{{< /note >}}
+
+Kamu harus menjalankan semua perintah dalam panduan ini sebagai `root`. Sebagai contoh, awali perintah 
+dengan `sudo`, atau masuk sebagai `root` dan kemudian baru menjalankan perintah sebagai pengguna `root`.
+
+### _Driver_ cgroup
+
+Ketika systemd dipilih sebagai sistem init untuk sebuah distribusi Linux, proses init menghasilkan
+dan menggunakan grup kontrol root (`cgroup`) dan proses ini akan bertindak sebagai manajer cgroup. Systemd memiliki integrasi yang ketat
+dengan cgroup dan akan mengalokasikan cgroups untuk setiap proses. Anda dapat mengonfigurasi
+_runtime_ Container dan kubelet untuk menggunakan `cgroupfs`. Menggunakan `cgroupfs` bersama dengan systemd berarti
+akan ada dua manajer cgroup yang berbeda.
+
+_Control group_ digunakan untuk membatasi sumber daya yang dialokasikan untuk proses.
+Sebuah manajer cgroup tunggal akan menyederhanakan pandangan tentang sumber daya apa yang sedang dialokasikan
+dan secara bawaan (_default_) akan memiliki pandangan yang lebih konsisten tentang sumber daya yang tersedia dan yang sedang digunakan. Ketika kita punya memiliki 
+dua manajer kita akan memiliki dua pandangan tentang sumber daya tersebut. Kita telah melihat kasus di lapangan
+di mana Node yang dikonfigurasi menggunakan `cgroupfs` untuk kubelet dan Docker, dan `systemd`
+untuk semua sisa proses yang berjalan pada Node maka Node tersebut akan menjadi tidak stabil dibawah tekanan sumber daya.
+
+Mengubah aturan sedemikian rupa sehingga _runtime_ Container dan kubelet kamu menggunakan `systemd` sebagai driver cgroup
+akan menstabilkan sistem. Silahkan perhatikan opsi `native.cgroupdriver=systemd` dalam pengaturan Docker di bawah ini.
+
+{{< caution >}}
+Mengubah driver cgroup dari Node yang telah bergabung kedalam sebuah Cluster sangat tidak direkomendasikan.
+Jika kubelet telah membuat Pod menggunakan semantik dari sebuah _driver_ cgroup, mengubah _runtime_ Container
+ke _driver_ cgroup yang lain dapat mengakibatkan kesalahan pada saat percobaan untuk membuat kembali PodSandbox
+untuk Pod yang sudah ada. Menjalankan ulang (_restart_) kubelet mungkin tidak menyelesaikan kesalahan tersebut. Rekomendasi yang dianjurkan
+adalah untuk menguras Node dari beban kerjanya, menghapusnya dari Cluster dan menggabungkannya kembali.
+
+{{< /caution >}}
+
+## Docker
+
+Pada setiap mesin kamu, installah Docker.
```

Comment:
rewording might be better
```suggestion
Pada setiap mesin kamu, mari menginstal Docker.
```

Reply:
fixed in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1